src/config.defaults.js

```diff
@@ -49,7 +49,8 @@ export default {
 
   // auth
   // eslint-disable-next-line prettier/prettier
-  jwtSecret: 'eAwI4zcTDd4Pvc8QtN9z57Fqsr4ENNcTpK1x4A1dCLj0Y44OravXZDzNbA-4VEwAIh1Hw3vn1nhB9ygWLqAGE4GiX6hjjLsJi8IJ',
+  jwtSecret:
+    'eAwI4zcTDd4Pvc8QtN9z57Fqsr4ENNcTpK1x4A1dCLj0Y44OravXZDzNbA-4VEwAIh1Hw3vn1nhB9ygWLqAGE4GiX6hjjLsJi8IJ',
```

Comment:
it has eslint ignore above, something about your settings or the new .prettierrc.json forcing it anyway?